sources/packages/backend/libs/test-utils/src/factories/sfas-application.ts

```diff
@@ -0,0 +1,40 @@
+import { SFASApplication } from "@sims/sims-db";
+import { getISODateOnlyString } from "@sims/utilities";
+import * as faker from "faker";
+
+/**
+ * Create fake SFAS application.
+ * @param options SFAS application options.
+ * - `initialValues` SFAS application initial values.
+ * @returns persisted SFAS application.
+ */
+export function createFakeSFASApplication(options?: {
+  initialValues?: Partial<SFASApplication>;
+}): SFASApplication {
+  const sfasApplication = new SFASApplication();
+  sfasApplication.id = faker.datatype.number({
+    min: 100000000,
+    max: 999999999,
+  });
+  sfasApplication.startDate =
+    options?.initialValues.startDate ??
+    getISODateOnlyString(faker.date.past(18));
+  sfasApplication.endDate =
+    options?.initialValues.endDate ?? getISODateOnlyString(faker.date.past(18));
+  sfasApplication.programYearId =
+    options?.initialValues.programYearId ?? 20222023;
+  sfasApplication.individualId = options?.initialValues.individualId ?? 1;
```

Comment:
Please use the individuals using relations property. 

![image](https://github.com/bcgov/SIMS/assets/54600590/c2c5e084-e36f-4b8f-8db0-bfe87964d292)